writing/org/OVERVIEW.md

```diff
@@ -0,0 +1,177 @@
+# Substrate
+
+An Open Architecture for Intelligence
+(or maybe ... An Open Architecture for Intelligent Systems?)
```

Comment:
"Intelligence" could well be interpreted as the kind of activity performed by The Intelligence Community — and in fact, the first search result for this phrase as is links to the defense sector.  But it's less of a mouthful than "Intelligent Systems", which for sure will be interpreted to mean something much more fundamental/general than we mean here (e.g., slime molds, starling murmurations, human beings).  I think we're better off leaning toward the "spies are cool" region of semantic space than the "physicists and mathematicians arguing about what intelligence means" region.